fabfile.py

```diff
@@ -445,8 +445,20 @@ def install_typewatch():
         local('cp jquery.typewatch.js {}'.format(dest_js_dir))
 
 
-# Tasks
+@recipe('fullcalendar.js')
+def install_fullcalendar_js():
+    with node_env():
```

Comment:
just fyi, you could also do it like this:

    with node_env(), lcd(os.path.join(env.ext_dir, 'fullcalendar'));